From 6e5d8cff72fb116d707d9cb165aec023c7fd3b27 Mon Sep 17 00:00:00 2001
From: Jean-Yves Avenard <jyavenard@mythtv.org>
Date: Thu, 19 Jun 2014 01:46:36 +1000
Subject: [PATCH 06/10] Fix dummy recording file sometimes not being deleted

This sometimes occurred at the start of LiveTV, causing a 0 byte recording that will never get deleted (and cause the frontend to keep trying to open recording for 10s)
---
 mythtv/libs/libmythtv/tv_rec.cpp |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/mythtv/libs/libmythtv/tv_rec.cpp b/mythtv/libs/libmythtv/tv_rec.cpp
index dcf4427..b4dfe50 100644
--- a/mythtv/libs/libmythtv/tv_rec.cpp
+++ b/mythtv/libs/libmythtv/tv_rec.cpp
@@ -4652,6 +4652,10 @@ bool TVRec::SwitchLiveTVRingBuffer(const QString & channum,
         curRecording = pginfo;
         SetRingBuffer(rb);
     }
+    else
+    {
+        delete rb;
+    }
 
     return true;
 }
-- 
1.7.10.2

